codex/uberdev-codex/skills/uberdev-cmd-review-pr/SKILL.md

```
@@ -684,13 +688,45 @@ Pass `--turbo` (anywhere in the arguments) to acknowledge invocation from `finis
+    match=re.fullmatch(r'(status|failure_class|signal_anchor):[ \t]*(.*)',line)
+    if not match: continue
+    key,value=match.groups()
+    if key in fields: raise SystemExit(2)
+    fields[key]=value.strip().strip('"').strip("'")
+if fields.get('status')=='AMBIGUOUS':
+    if fields.get('failure_class') not in {None,'','null'}: raise SystemExit(2)
+    print('flaky\t'); raise SystemExit(0)
+classes={'code_bug','billing_quota','platform_outage','flaky','env_drift','stale_base'}
+anchor=fields.get('signal_anchor','')
+if fields.get('status')!='CLASSIFIED' or fields.get('failure_class') not in classes: raise SystemExit(2)
+if not re.fullmatch(r'(?:gh-run-[1-9][0-9]*|[A-Za-z0-9._/-]+):[1-9][0-9]*',anchor): raise SystemExit(2)
```

markdownlint-cli2 (0.23.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "0-9"

(MD052, reference-links-images)